app/models/aspace_repository.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+
+# Fetches and transforms repository information from archivesspace
+class AspaceRepository
+  HARVESTABLE_REPOSITORIES = %w[ars cubberley eal].freeze
```

Comment:
My only thought about this is whether it should be kept in a more obvious config location. But if we think the list will eventually be pretty fixed, then maybe not necessary.

Reply:
I had similar thoughts, but wanted to defer setting up anything more complicated until we know more about how we want to manage these.